src/main/java/core/basesyntax/ReadWriteList.java

```diff
@@ -10,16 +10,33 @@ public class ReadWriteList<E> {
     private ReadWriteLock lock = new ReentrantReadWriteLock();
 
     public void add(E element) {
-        // write your code here
+        lock.writeLock().lock();
+        try {
+            list.add(element);
+        } finally {
+            lock.writeLock().unlock();
+        }
     }
 
     public E get(int index) {
-        // write your code here
-        return null;
+        lock.readLock().lock();
+        E element;
+        try {
+            element = list.get(index);
+        } finally {
+            lock.readLock().unlock();
+        }
+        return element;
```

Comment:
```suggestion
        lock.readLock().lock();
        try {
            return element = list.get(index);
        } finally {
            lock.readLock().unlock();
        }
```
You can simplify it.